.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
```

Comment:
I'm unsure why you added an editorconfig file.

I understand it's useful but I'm unsure what issue you were trying to solve.

People using editor that supports editorconfig will have an instant benefit, but people who don't will have no clue there is a style to comply with

I would like to recommend to add a GHA to check files comply with the conf

https://github.com/editorconfig-checker/editorconfig-checker

Also, I'm a bit surprised to see this change added out of the blue in a PR about a feature.